### ISSUE TITLE
Change deis IMAGE_PULL_POLICY setting to "Always"

### ISSUE THREAD

**Issue by jgmize** [mozmeao/infra#447]
We ran into some issues in the Tokyo cluster where the integration tests were failing due to old images being used on new deployments, preserving the previous behavior. In order to force new images to be deployed, I changed the imagePullPolicy from the default "IfNotPresent" to Always.

https://kubernetes.io/docs/concepts/configuration/overview/#container-images
https://github.com/deis/controller/blob/master/rootfs/api/settings/production.py#L270

**Comment by pmac** [comment]
Thanks! Will this be applied to all k8s clusters?

**Comment by jgmize** [comment]
I'd like to implement https://github.com/mozilla/bedrock/issues/5067 first to confirm the fix and identify if it's actually needed for the other clusters. My current hypothesis is that the reason we ran into this issue is unique not only to the tokyo cluster, but also to the bedrock-dev app on it. Specifically, in order to resolve a separate issue I had to delete and recreate the app, which caused the revisions to start over at 1, which resulted in docker tag re-use due to some deis internal implementation details that I'd be happy to go into if you're curious, but we should probably focus on #448 instead.
